More/Delegation/Tests/DelegationTest.php

```diff
@@ -7,11 +7,25 @@
 
 class DelegationTest extends TestCase
 {
-    public function testHowTeamLeadWriteCode()
+    /**
+     * @test
+     */
+    public function teamLeadCanBlameJuniorForBadCode()
```

Comment:
please use the test in method name convention here instead of using `@test`: `testTeamLeadCanBlameJuniorForBadCode`